ui/packages/catalog-ui-search/src/main/webapp/component/confirmation/query/confirmation.query.hbs

```diff
@@ -34,7 +38,7 @@
         </button>
         <button class="confirmation-new is-negative">
             <span>
-                Create New Workspace
+                Create New Workspace from Search
```

Comment:
Think this button is too big and doesn't match the rest of our styling.

Reply:
I don't disagree, but this is coming from our UX experts. I wonder if future changes are planned for the UI as a whole that match this styling. At the very least, maybe we could shrink the size of the font in the button. Thoughts @TeresaHudson?